crates/amaru/src/stages/mod.rs

```diff
@@ -369,7 +369,26 @@ impl AsTip for Header {
 
 #[cfg(test)]
 mod tests {
-    use super::{bootstrap, Config, StorePath::*};
+    use std::path::PathBuf;
+
+    use amaru_kernel::network::NetworkName;
+
+    use super::{
+        bootstrap, Config,
+        StorePath::{self, *},
+    };
+
+    fn default_config() -> Config {
+        Config {
+            ledger_store: StorePath::OnDisk(PathBuf::from("./ledger.db")),
+            chain_store: StorePath::OnDisk(PathBuf::from("./chain.db.1")),
```

Comment:
`.1` ?

Reply:
I just moved the code so it was already this way.